src/geometry/SphereSceneNode.cxx

```diff
@@ -214,10 +191,20 @@ void SphereLodSceneNode::kill()
 }
 
 
-SphereLodSceneNode::SphereLodSceneNode(const GLfloat pos[3], GLfloat _radius) :
-    SphereSceneNode(pos, _radius),
+SphereSceneNode::SphereSceneNode(const GLfloat pos[3], GLfloat _radius) :
```

Comment:
why is this down here? Shouldn't methods in a file be in the same order as the declarations in the header? Random code organization isn't helpful?

Reply:
I left it where it was. Just changed the name. If I reorder you get lost with the changes